test/e2e/tests/tokens/custom-token-add-approve.spec.js

```diff
@@ -52,14 +44,12 @@ describe('Create token, approve token and approve token without gas', function (
         await driver.waitForSelector(
           '[data-testid="import-tokens-modal-custom-decimals"]',
         );
-        await driver.delay(2000);
 
         await driver.clickElement({
           text: 'Next',
           tag: 'button',
         });
 
-        await driver.delay(2000);
```

Comment:
we don't need these dalys, since the clickElement method already has logic in place for finding clickable elements in the first place